ui/dashboards/src/components/AddPanel/AddPanel.tsx

```diff
@@ -28,18 +28,16 @@ import { Drawer, ErrorAlert } from '@perses-dev/components';
 import { JsonObject } from '@perses-dev/core';
 import { PluginBoundary } from '@perses-dev/plugin-system';
 import { ChangeEvent, FormEvent, useState } from 'react';
-import { useLayouts, usePanels } from '../../context';
+import { useDashboardApp, useLayouts, usePanels } from '../../context';
 import { removeWhiteSpacesAndSpecialCharacters } from '../../utils/functions';
 import { PanelOptionsEditor, PanelOptionsEditorProps } from './PanelOptionsEditor';
 
-interface AddPanelProps {
```

Comment:
Removed these props and storing these states in zustand instead because there are a few places in the app where you can open the add panel drawer